kuma/wiki/models.py

```diff
@@ -1358,22 +1322,10 @@ def redirect_url(self):
                 if len(url) > 1:
                     if url.startswith(settings.SITE_URL):
                         return url
-                    elif (url[0] == '/' and url[1] != '/'):
+                    elif url[0] == '/' and url[1] != '/':
                         return url
-                elif (len(url) == 1 and url[0] == '/'):
+                elif len(url) == 1 and url[0] == '/':
                     return url
-                else:
-                    return None
```

Comment:
Okay so long as it's a language construct and not part of a library or something.

/me hates magic